dspace-xmlui/src/main/java/org/dspace/app/xmlui/objectmanager/ItemAdapter.java

```diff
@@ -653,6 +648,16 @@ else if (dso.getType() == Constants.ITEM)
         endElement(METS,amdSecName);
     }
 
+    private Element disseminateElement(DisseminationCrosswalk crosswalk, DSpaceObject dso) throws CrosswalkException, IOException, SQLException, AuthorizeException {
+        Element dissemination;
+        if (crosswalk instanceof METSRightsCrosswalk) {
+            dissemination = ((METSRightsCrosswalk)crosswalk).disseminateElement(context,dso);
```

Comment:
I'm worried about the fact that this new `disseminateElement(context,dso)` method is ONLY used by XMLUI.  While that may be an appropriate change for this bug fix, it means that this method will never be called when a Curation Task runs from CLI or from JSPUI or similar.